apps/condo/domains/acquiring/schema/CreatePaymentByLinkService.js

```diff
@@ -36,7 +36,7 @@ const ERRORS = {
         code: BAD_USER_INPUT,
         type: ALREADY_EXISTS_ERROR,
         message: 'Provided receipt already paid',
-        messageForUser: 'api.billing.error.alreadyPaid',
+        messageForUser: 'api.billing.billingReceipt.RECEIPT_ALREADY_PAID_ERROR',
```

Comment:
```suggestion
        messageForUser: 'api.billing. createPaymentByLink.RECEIPT_ALREADY_PAID_ERROR',
```

seems like it's appropriate to use the mutation name, not the model

Reply:
this error translation is being used in different apps and two mutation in condo, so i think i'ts easier to write 'billingReceipt', than determine which mutation is more correct for this error